cli/test/puppeteer.integration.js

```diff
@@ -28,6 +29,27 @@ describe('puppeteer', () => {
     assert.equals(json.tests[0].fullTitle, 'test passes');
   });
 
+  it('reads from stdin', async () => {
+    let result;
+    try {
+      const cp = execa('../../index.js', ['--driver', 'puppeteer', '-'], {
```

Comment:
Alternatively we could also teach `run` about this behaviot, dunno if it's worth it?

Reply:
That'd make it rather complicated, no?